editor/plugins/script_editor_plugin.cpp

```diff
@@ -866,6 +873,11 @@ void ScriptEditor::_menu_option(int p_option) {
 				debugger->set_hide_on_stop(visible);
 			debug_menu->get_popup()->set_item_checked(debug_menu->get_popup()->get_item_index(DEBUG_SHOW_KEEP_OPEN), !visible);
 		} break;
+		case DEBUG_OPEN_WITH_EXTERANL_EDITOR: {
```

Comment:
Typo: EXTERANL -> EXTERNAL

While amending the commit you can also fix the typo in "expirence" ;)